### PR TITLE
async properties

### PR DESCRIPTION
**Pull request by debris**
added: 
- async way of getting properties

removed:
- property setter (it was unused)

### CODE REVIEW

File: dist/web3.js [hunk 1]
```diff
@@ -3211,15 +3218,20 @@ Property.prototype.get = function () {
...
- * @method set
- * @param {Object} new value of the property
+ * @method asyncGet
+ * @param {Function}
  */
-Property.prototype.set = function (value) {
-    return RequestManager.getInstance().send({
-        method: this.setter,
-        params: [this.formatInput(value)]
+Property.prototype.asyncGet = function (callback) {
```
Comment (frozeman) [inline dist/web3.js]: Though i would habe called it `getAsync`

Comment (debris) [inline dist/web3.js]: fixed

**Comment by frozeman** [comment]
I really like the changes, but i makes myself question the whole use of "properties". Why we don't plainly use:

```js
// sync
var res = web3.eth.blockNumber();

// async
web3.eth.blockNumber(function(e,res) { ... })
```

You can still use the properties synchronously, but you need to add a `()`. It at least makes everything more unified. Also in terms of the docs.

**Comment by debris** [comment]
but we will break another thing then :) I would prefer to do it later, after merging this pr and releasing new version :)

**Comment by frozeman** [comment]
ok. lets do it later then. it will anyway be very much recommended to use only async, except you know what you're doing.
so we can later get "rid" of the pure property ones..

**Pull request merged**